src/adapters/RoomsAdapter.js

```diff
@@ -0,0 +1,46 @@
+import {throwError} from 'rxjs';
+
+import WebexAdapter from './WebexAdapter';
+
+/**
+ * This is a base class that defines the interface that maps room data.
+ * Developers that want to extend `RoomsAdapter` must implement all of its methods,
+ * adhering to the exact parameters and structure of the returned objects.
+ */
+export default class RoomsAdapter extends WebexAdapter {
```

Comment:
Document the `Room` object just after the class, like we do on the other adapters.
(See [`Person` in `PeopleAdapter`](https://github.com/webex/components/blob/master/src/adapters/PeopleAdapter.js#L31) and [`Activity` in `ActivitiesAdapter`](https://github.com/webex/components/blob/master/src/adapters/ActivitiesAdapter.js#L12))

Reply:
I came up with something for now

Reply:
Looks good